usaspending_api/api_contracts/contracts/v2/search/spending_by_transaction.md

```diff
@@ -1,7 +1,7 @@
 FORMAT: 1A
 HOST: https://api.usaspending.gov
 
-# IDV Funding [/api/v2/search/spending_by_transaction/]
+# Funding [/api/v2/search/spending_by_transaction/]
```

Comment:
Could you please rename this to `Spending by Transaction` instead of just `Funding`. Looks like a similar endpoint `/api/v2/search/spending_by_transaction_count/` has the same problem. Can you go ahead and update the title to `Spending by Transaction Count`?